crates/bevy_text/src/text.rs

```diff
@@ -170,3 +173,28 @@ impl Default for TextStyle {
         }
     }
 }
+
+/// Determines how lines will be broken when preventing text from running out of bounds.
+#[derive(Debug, Clone, Copy, PartialEq, Eq, Hash, Reflect, Serialize, Deserialize)]
+#[reflect(Serialize, Deserialize)]
+pub enum TextLineBreakBehaviour {
+    /// Lines will be broken up at the nearest word boundary, usually at a space.<br/>
+    /// This behaviour suits most cases, as it keeps words intact. Aims to implement the Unicode line breaking algorithm.
+    Unicode,
```

Comment:
```suggestion
    WordBoundary,
```

Even if it is not completely accurate, naming it `WordBoundary` is clearer for someone just looking at the enum variant names. A link to https://www.unicode.org/reports/tr14/ (RFC for the unicode line breaking algorithm) would also be nice